src/test/scala/is/hail/methods/gqDpStatsSuite.scala

```diff
@@ -1,69 +0,0 @@
-package is.hail.methods
-
-import is.hail.SparkSuite
-import is.hail.utils.{TestRDDBuilder, _}
-import is.hail.variant.Variant
-import org.testng.annotations.Test
-
-class gqDpStatsSuite extends SparkSuite {
```

Comment:
We now test aggregators against variant / sample qc. If sample/variant qc == R results, and aggregators == v/s qc, then we can remove the first test and as long as we don't change both at once, it'll stay correct.

Mostly I wanted to remove the "results" method, without reworking this whole test.